CMakeLists.txt

```
@@ -9,6 +9,7 @@ set(PACKAGE_NAME  "HyKKT")
 set(PACKAGE_TARNAME "hykkt")
 
 option(HYKKT_TEST_WITH_BSUB "Use `jsrun` instead of `mpirun` commands when running tests" OFF)
+option(HYKKT_USE_AMD "Use AMD from SuiteSparse" OFF)
```

```suggestion
option(HYKKT_USE_AMD_ORDERING "Use AMD from SuiteSparse" OFF)
```
Specify that "AMD" stands for approximate minimum degree ordering; not for AMD GPUs.

---

Can HyKKT eventually be used with AMD GPUs?

---

HyKKT does not build on AMD GPU platforms as yet, in this context AMD is Approximate Minimum Degree library from SuiteSparse.